renderer/src/hooks/StationRewards.tsx

```diff
@@ -8,9 +8,10 @@ type ScheduledRewardsResponse = {
 }[]
 
 async function getHistoricalScheduledRewards (address: string) {
+  const yesterday = new Date(new Date().setDate(new Date().getDate() - 1))
```

Comment:
So many new Date instances! Have you considered using something like this? 

```js
const yesterday = new Date(Date.now() - 24 * 3600_000)
```

Reply:
I don't have a strong opinion on this. To me the current version is better because
- performance doesn't matter here at all
- it uses `Date` APIs that work on a day level, not on a millisecond level
- it doesn't use magics or constants that you need to remember (3600s = 1h)
What is your concern with the current version? Or are you saying it's difficult to read because of the many date instances?

Reply:
I find Date APIs confusing - some values are zero-based (months), and some are one-based (day of the month). `getDate` returns the day of the month, and `getTime` returns milliseconds since the Unix epoch. I always have to check the docs to recall what each method does.

Anyhow, this is not a big deal. The variable name clearly communicates what we are calculating, the rest is an implementation detail.

Reply:
I understand! I also feel that way about 0 vs 1 based Date functions, but I feel safe here since we're passing a value + 1, after reading it from Date, so whether the function is 0 based or 1 based doesn't make a difference